src/pages/dmt5/_flexibility.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Flex, SectionContainer } from 'components/containers'
+import { Header, Text } from 'components/elements'
+import { deriv_app_url } from 'common/utility'
+import { Localize, localize } from 'components/localization'
+import SyntheticIcon from 'images/svg/synthetic.svg'
+import FinancialIcon from 'images/svg/financial.svg'
+import FinancialStpIcon from 'images/svg/financial-stp.svg'
+
+const content = [
+    {
+        header: <Localize translate_text="Synthetic" />,
+        text: (
+            <Localize translate_text="Trade CFDs on our exclusive, proprietary synthetic indices 24/7 which simulate real-world market movements." />
+        ),
+        icon: <SyntheticIcon />,
+    },
+    {
+        header: <Localize translate_text="Financial" />,
+        text: (
+            <Localize translate_text="Trade forex, commodities, cryptocurrencies, major (standard and micro-lots), and minor currency pairs on high leverage." />
+        ),
+        icon: <FinancialIcon />,
+    },
+    {
+        header: <Localize translate_text="Financial STP" />,
+        text: (
+            <Localize translate_text="Trade major, minor, and exotic currency pairs with tight spreads and higher trade volumes, straight to the market." />
+        ),
+        icon: <FinancialStpIcon />,
+    },
+]
+const Section = styled(SectionContainer)`
+    display: flex;
+    padding: 8rem 12rem;
+    justify-content: center;
+    flex-direction: column;
+`
+const ClientCard = styled.article`
+    margin: 0 0 0 2rem;
+    background-color: var(--color-white);
+    border-radius: 4px;
+    box-shadow: 0 4px 8px 0 rgba(14, 14, 14, 0.1);
+    width: 38.4rem;
+    padding: 3.2rem 2.4rem;
+    height: 100%;
+    position: relative;
+`
+const ExternalLink = styled.a`
```

Comment:
please use `LocalizedLink` for the external Links and pass `external` as the props :pray: